applications/robot-motion-control/include/robot1_conf.hpp

```diff
@@ -46,7 +46,7 @@ constexpr float angular_speed_pid_kd = 0;
 // Linear threshold
 constexpr float linear_threshold = 1;
 // Angular threshold
-constexpr float angular_threshold = 1;
+constexpr float angular_threshold = 2;
```

Comment:
Maybe you can verify this point during real testing session.

I think we should be able to maintain control loop accurate as before.

Maybe try this two configuration and check the impact in robot stability:
- motion control engine (Odometry) @100Hz -> postion loop @20Hz -> Speed loop @100Hz (~5 ratio)
- motion control engine (Odometry) @200Hz-> postion loop @20Hz -> Speed loop @200Hz (~10 ratio)

The goal is to keep position control loop as faster as before. My assumption is that 100ms or 200ms is too high to preserve robot accuracy and faster response time.